lib/std/unicode.zig

```diff
@@ -770,6 +769,48 @@ fn calcUtf16LeLen(utf8: []const u8) usize {
     return dest_len;
 }
 
+/// Print the given `utf16le` string
+fn formatUtf16le(
+    utf16le: []const u16,
+    comptime fmt: []const u8,
+    options: std.fmt.FormatOptions,
+    writer: anytype,
+) !void {
+    const unknown_codepoint = 0xfffd;
+    _ = fmt;
+    _ = options;
+    var buf: [300]u8 = undefined; // just a random size I chose
+    var it = Utf16LeIterator.init(utf16le);
+    var u8len: usize = 0;
+    while (it.nextCodepoint() catch unknown_codepoint) |codepoint| {
+        u8len += utf8Encode(codepoint, buf[u8len..]) catch
```

Comment:
Note that this may not be *possible*.  If we get a codepoint from decoding it, it *might* be impossible for it to fail in `utf8Encode`.  Someone who knows more might be able to clarify.